src/types/profile.js

```diff
@@ -817,6 +823,9 @@ export type ProfileMeta = {|
   // "unknown" state.  For now we don't do much with it but we may want to
   // propose a manual symbolication in the future.
   symbolicated?: boolean,
+  // A boolean flag indicating that symbolication is not supported, default false
```

Comment:
Default is currently undefined. We can either say something like "default is undefined", or we can just remove that part. 
For this line this looks enough to me, since we explain it's used for imported profiles in the line below:
```
// A boolean flag indicating that symbolication is not supported.
```